manifests/profile/www_lib/vhosts/www_lib.pp

```diff
@@ -21,16 +21,9 @@
     servername                    => "${prefix}www.${domain}",
     ssl                           => true,
     usertrack                     => true,
-    cosign                        => true,
+    cosign                        => false,
     docroot                       => $docroot,
     directories                   => [
-      {
```

Comment:
Removing this is not related to cosign, but rather addressing an oversight when I cleaned up the www.lib vhost for fallback duty last year. There is no CGI within the current library website static build.